src/assets/lang/fr.json

```diff
@@ -23,9 +23,14 @@
   "betweenValues": "Entre {startValue} et {stopValue}",
   "overValue": "Plus de {startValue}",
 
-  "entityParagraph" : "Cette {variableLink} s'applique aux {entityLink}",
+  "entityParagraph" : "Cette {variableLink} s'applique à l'{entityLink}",
+  "entityRolesParagraph": "Cette {entityLink} définit les rôles suivants&nbsp;:",
```

Comment:
Doesn't work:


![image](https://user-images.githubusercontent.com/11834997/46259655-80598600-c4dc-11e8-9590-74e2e58036be.png)


(`allowedValues ` further down doesn't work either)

To allow HTML in formatted messages, you need to use `FormattedHTMLMessage` instead of `FormattedMessage`

Reply:
[FormattedHTMLMessage](https://github.com/yahoo/react-intl/wiki/Components#formattedhtmlmessage) is not recommended (legacy) and it doesn't work with `values`.
Tried something without `&nbsp;` but preserving spaces : `<pre>` 😬